assessment.md

```diff
@@ -56,14 +56,76 @@ Project Management is Covered by Team 2
 ### Foundations of Software Engineering
 
 * Requirements engineering
+  * :disappointed: No effort to change refine or contribute to documentation
+  * :confused: Very short or vague description of documentation
+  * :neutral_face: Conveys information accurately and descriptively enough for
+  everyone to understand
+  * :smiley: Well thought-out and descriptive documentation and definitions so
+  everyone can be well informed
+  * :sunglasses: A very structured and detailed outlining of all of the elements
+  in a software system
 * Software architecture
+  * :disappointed: No carefully thought-out contribution to the layout of the Project
+  * :confused: Very little amount of shape to the architecture to the Project
+  * :neutral_face: There is some sort of plan for the system
+  * :smiley: There is a well thought-out plan for the system
+  * :sunglasses: There is a very detailed and complete plan laid out for the system
 * Software design
+  * :disappointed: No effort to make a design
+  * :confused: Small effort made to put in place a layout of a design system
+  * :neutral_face: Some small designs are able to be seen so a solution is visible
+  * :smiley: A solution that is rich in coding layout and architectural design
+  * :sunglasses: All expectations have been exceeded with a well thought-out high
+  level software solution with both extremely detailed algorithmic and architectural
+  designs
 * Software documentation
+  * :disappointed: No written documentation or docstrings of how the code works
+  * :confused: Very little or unclear documentation or docstrings in the code
+  of how the code works
+  * :neutral_face: Software documentation or docstrings have a decent bit of
+  detail describing the code and how the program will run
+  * :smiley: In-depth detailed documentation or docstrings within the project
+  that explain the reasoning behind the code
+  * :sunglasses: Very in-depth detailed explanation of code with use of
+  documentation or docstrings to show and explain how the code will be used
```

Comment:
@Cory-Wiard I suggest that you rephrase "the use of documentation or docstrings" since documentation should include docstrings, comments, README, etc. There are also different styles of documentation. The wording here gives a sense of picking a strategy between either documentation or docstring, which is incorrect.